sections/shared/modals/SelectCurrencyModal/SelectCurrencyModal.tsx

```diff
@@ -153,17 +159,17 @@ export const SelectCurrencyModal: FC<SelectCurrencyModalProps> = ({
 				{synthsWalletBalancesQuery.isLoading ? (
 					<Loader />
 				) : synthsResults.length > 0 ? (
-					synthsResults.map((synth) => {
+					synthsResults.map((synth: any) => {
```

Comment:
maybe just add a TODO to add better typings in the future here

Reply:
done